test/ConnectionTest.cpp

```diff
@@ -7,6 +7,8 @@
 
 namespace {
 
+using namespace Origin;
```

Comment:
Be consistent. In some places you writd full path, but here you write using. By the way, avoid writing using namespace. Write exactly what youre going to use if full path is not an option for you